infra/packages/shared/src/index.ts

```
@@ -42,3 +42,101 @@ export const CLOUDWATCH_KINESIS_STREAM_ROLE_ARN =
+  [COMMS_SERVICE_URL]: 'https://comms-service.macro.com',
+  [EMAIL_SERVICE_URL]: 'https://email-service.macro.com',
+  [PROPERTIES_SERVICE_URL]: 'https://properties-service.macro.com',
+  [METERING_SERVICE_URL]: 'https://metering.macro.com',
+  [STATIC_FILE_SERVICE_URL]: 'https://static-file-service.macro.com',
+  [ORGANIZATION_SERVICE_URL]: 'https://organization-service.macro.com',
+  [NOTIFICATION_SERVICE_URL]: 'https://notifications.macro.com',
+  [AUTHENTICATION_SERVICE_URL]: 'https://auth-service.macro.com',
+  [DOCUMENT_STORAGE_SERVICE_URL]: 'https://cloud-storage.macro.com',
+  [CONNECTION_GATEWAY_URL]: 'https://connection-gateway.macro.com',
+  [DOCUMENT_COGNITION_SERVICE_URL]: 'https://document-cognition.macro.com',
+  [LEXICAL_SERVICE_URL]: 'https://lexical-service-prod.macroverse.workers.dev',
+  [SEARCH_SERVICE_URL]: 'https://search-service.macro.com',
+}
+
+const DEV_URLS = {
+  [SYNC_SERVICE_URL]: 'https://sync-service-dev3.macroverse.workers.dev',
+  [COMMS_SERVICE_URL]: 'https://comms-service-dev.macro.com',
+  [EMAIL_SERVICE_URL]: 'https://email-service-dev.macro.com',
+  [PROPERTIES_SERVICE_URL]: 'https://properties-service-dev.macro.com',
+  [METERING_SERVICE_URL]: 'https://metering-dev.macro.com',
+  [STATIC_FILE_SERVICE_URL]: 'https://static-file-service-dev.macro.com',
+  [ORGANIZATION_SERVICE_URL]: 'https://organization-service-dev.macro.com',
+  [NOTIFICATION_SERVICE_URL]: 'https://notifications-dev.macro.com',
+  [AUTHENTICATION_SERVICE_URL]: 'https://auth-service-dev.macro.com',
+  [DOCUMENT_STORAGE_SERVICE_URL]: 'https://cloud-storage-dev.macro.com',
+  [CONNECTION_GATEWAY_URL]: 'https://connection-gateway-dev.macro.com',
+  [DOCUMENT_COGNITION_SERVICE_URL]: 'https://document-cognition-dev.macro.com',
+  [LEXICAL_SERVICE_URL]: 'https://lexical-service-dev.macroverse.workers.dev',
+  [SEARCH_SERVICE_URL]: 'https://search-service-dev.macro.com',
+}
+
+const URLS = stack === STACK_PROD ? PROD_URLS : DEV_URLS;
```

This will break if we ever add a staging environment. I think URLS should be a function that can throw an error if you provide an unexpected stack name.

---

see signal for defining cost